File: fix.py
import sys

with open("src/data/articles.js", "r", encoding="utf-8") as f:
    content = f.read()

start_idx = content.find('    {\n        id: "haribhakta-putali-rani"')
end_idx = content.find('    },\n    {\n        id: 1,')

if start_idx != -1 and end_idx != -1:
    article_str = content[start_idx:end_idx + 6]
    content = content[:start_idx] + content[end_idx + 6:]
    last_bracket = content.rfind(']')
    content = content[:last_bracket] + ',\n' + article_str + '\n];\n'
    
    with open("src/data/articles.js", "w", encoding="utf-8") as f:
        f.write(content)
    print("Success")
else:
    print("Failed to find boundaries")

File: fix.js
const fs = require('fs');
let content = fs.readFileSync('src/data/articles.js', 'utf8');

// Fix syntax error at the top
content = content.replace('    t const articles = [\n', '');

// Find the Haribhakta article
const startObj = content.indexOf('    {\n        id: "haribhakta-putali-rani"');
const endObj = content.indexOf('    },\n    {\n        id: 1,');

if (startObj !== -1 && endObj !== -1) {
    const articleStr = content.slice(startObj, endObj + 6);
    // Remove it from the top
    content = content.slice(0, startObj) + content.slice(endObj + 6);
    
    // Append it to the bottom
    const lastBracket = content.lastIndexOf(']');
    content = content.slice(0, lastBracket) + ',\n' + articleStr + '\n];';
}

fs.writeFileSync('src/data/articles.js', content);
